lib/split/experiment.rb

```diff
@@ -43,6 +46,11 @@ def set_alternatives_and_options(options)
       self.metadata = options_with_defaults[:metadata]
       self.friendly_name = options_with_defaults[:friendly_name] || @name
       self.retain_user_alternatives_after_reset = options_with_defaults[:retain_user_alternatives_after_reset]
+
+      if self.algorithm == Split::Algorithms::SystematicSampling
+        self.cohorting_block_seed = options_with_defaults[:cohorting_block_seed] || self.name.to_i(36)
```

Comment:
Default seed if none is specified in YAML is based on the name of the experiment: `self.name.to_i(36)`

I think something like:
```ruby
Digest::MD5.hexdigest('exp893_solo').to_i(16)
=> 214402640715795185602270768195163573728
```

Would be better, but I wasn't sure if importing Digest module would be overkill

Reply:
Just want to point out, `to_i` doesn't support special characters, and it may also end up with a really long integer. How about something like https://apidock.com/ruby/v2_5_5/String/sum ?